dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoFactory.java

```diff
@@ -45,6 +45,7 @@ public static SourceCodeRepoInterface createGitHubAppRepo(String token) {
         return new GitHubSourceCodeRepo("JWT", token);
     }
 
+    // Assumes the token has already been refreshed
```

Comment:
Nitpick: Might as well use Javadoc style for these two comments in case we mouse-over tooltip these in the IDE